### ISSUE TITLE
_kCFStreamErrorDomainWinSock not found in CoreServices.framework

### ISSUE THREAD

**Issue by kayblaurock** [darlinghq/darling#873]
**Test Application**
[AdobeUpdateServerSetupTool](https://helpx.adobe.com/en/enterprise/admin-guide.html/enterprise/using/setup-ausst.ug.html) (Command line program from the macOS AUSST package)

**Expected Result**
Program runs without issue on macOS.

**Actual Result**
Program doesn't run, instead we get an dyld error about a missing symbol.

**Steps To Reproduce**
1. build Darling from source
2. run AdobeUpdateServerSetupTool from the Darling shell.
3. on Darling but NOT macOS we get the following error:
```
Darling [/Volumes/SystemRoot/usr/local/AUSST]$ ls -al
total 33880
drwxr-xr-x   2 administrator  administrator     4096 17 Sep 19:23 .
drwxr-xr-x  12 administrator  administrator     4096 17 Sep 19:24 ..
-rwxr-xr-x   1 administrator  administrator  1190208 17 Sep 20:57 AdobeUpdateServerSetupTool
-rwxr-xr-x   1 administrator  administrator  7838368 17 Sep 20:57 HDPIM.dylib
-rwxr-xr-x   1 administrator  administrator  6413616 17 Sep 20:57 HUM.dylib
-rwxr-xr-x   1 administrator  administrator  1892352 17 Sep 20:57 unar
Darling [/Volumes/SystemRoot/usr/local/AUSST]$ ./AdobeUpdateServerSetupTool 
dyld: Symbol not found: _kCFStreamErrorDomainWinSock
  Referenced from: /Volumes/SystemRoot/usr/local/AUSST/AdobeUpdateServerSetupTool
  Expected in: /System/Library/Frameworks/CoreServices.framework/Versions/A/CoreServices
 in /Volumes/SystemRoot/usr/local/AUSST/AdobeUpdateServerSetupTool
abort_with_payload: reason: Symbol not found: _kCFStreamErrorDomainWinSock
  Referenced from: /Volumes/SystemRoot/usr/local/AUSST/AdobeUpdateServerSetupTool
  Expected in: /System/Library/Frameworks/CoreServices.framework/Versions/A/CoreServices
 in /Volumes/SystemRoot/usr/local/AUSST/AdobeUpdateServerSetupTool; code: 4
Abort trap: 6
Darling [/Volumes/SystemRoot/usr/local/AUSST]$ 
```
This seems strange to me, because that symbol is located in CFNetwork.framework. So why does dyld expect to find it in CoreServices.framework?
Any hints to work around or solve the issue are greatly appreciated! :-)

**System Information**
Darling: Debian Bullseye (Testing, as of 2020-09-17) 
macOS: 10.15.6

| Software | Version |
| --- | --- |
| Darling | 2020-09-17 |
| AUSST | 2020-05-21 |

**Comment by HinTak** [comment]
Many of the big frameworks re-exports symbols from other sub-frameworks. In this case, I think it is possible that CoreSevices re-exports CFNetwork' symbols, and client apps have a choice of linking between the two.

There are a few places in darling which try to the same re-exports as mac os x. You can find them and use them as hints on how to fix yours. (I am just a regular darling user, who have done my share of looking and fixing issues that mattered to me - my opinion should be taken as such)

**Comment by facekapow** [comment]
Yes, @HinTak is right; CoreServices re-exports CFNetwork. This is simply an issue of `kCFStreamErrorDomainWinSock` not being defined in CFNetwork. Should be simple enough to fix; just define `kCFStreamErrorDomainWinSock` in CFNetwork.
